src/meta/src/hummock/manager/tests.rs

```diff
@@ -2044,6 +2044,8 @@ async fn test_move_tables_between_compaction_group() {
     assert_eq!(branched_ssts.len(), 2);
 }
 
+// TODO: re-enable this test after solving global metric issue when running test in multi-thread.
```

Comment:
👀 Shall we fix this or create an issue for it?

Reply:
I'll fix it.